docs/faq/subgraphAPI.md

```diff
@@ -0,0 +1,99 @@
+## How to Integrate Backend Libraries to MXNet with the Subgraph API
+
+The subgraph API has been proposed and implemented as the default mechanism for integrating backend libraries to MXNet. The subgraph API is a very flexible interface. Although it was proposed as an integration mechanism, it has been used as a tool for manipulating NNVM graphs for graph-level optimizations, such as operator fusion.
+
+The subgraph API works as the following steps:
+
+* Search for particular patterns in a graph.
+* Group the operators/nodes with particular patterns into a subgraph and shrink the subgraph into a single node.
+* Put the subgraph node back to the original graph to replace the subgraph.
```

Comment:
Rephrase for clarity.
How about, Replace the subgraph with original subgraph node.